project/PATCH-RELEASES.md

```diff
@@ -0,0 +1,53 @@
+# Docker patch (bugfix) release process
+
+Patch releases (the 'Z' in vX.Y.Z) are intended to fix major issues in a
+release. Docker open source projects follow these procedures when creating a
+patch release;
+
+After each release (both "major" (vX.Y.0) and "patch" releases (vX.Y.Z)), a
+patch-release milestone (vX.Y.Z + 1) is created.
```

Comment:
@tiborvass I like that

(also saw some other milestones that may be interesting, e.g. the X.Y-maybe milestone is used for "features" that *may* be included in a release, but undecided)